hello-server.php

```diff
@@ -45,74 +43,184 @@
  *
  * PHP version 5.6-7.1
  *
- *
- * @package Hello_Server
+ * @package    Hello_Server
+ * @version    1.2.1
  * @author     Dan Pock <dpock@liquidweb.com>
  * @copyright  2017 Liquid Web
  * @license    http://www.gnu.org/licenses/gpl-2.0.txt  GPL-2.0
  * @link       https://github.com/liquidweb/hello-server
  */
 
-function hello_server_get_info() {
-	$info = get_transient('hello_server_info_cache_'.$_SERVER["SERVER_ADDR"]);
-	if (false === $info) {
-		// It wasn't there, so regenerate the data and save the transient
-		$info = [
-			'ip' => get_server_ip(),
-			'hostname' => gethostname()
-		];
-		set_transient('hello_server_info_cache_'.$_SERVER["SERVER_ADDR"], $info, 24 * HOUR_IN_SECONDS);
+
+/**
+ * Load our textdomain for internationalization support.
+ *
+ * @return void
+ */
+function lw_load_plugins_textdomain() {
+	load_plugin_textdomain( 'hello-server', false, dirname( dirname( plugin_basename( __FILE__ ) ) ) . '/languages/' );
+}
+add_action( 'plugins_loaded', 'lw_load_plugins_textdomain' );
+
+/**
+ * Pull our server info for display use.
+ *
+ * @return array $info  The IP and hostname being displayed.
+ */
+function lw_hello_server_get_info() {
+
+	// Bail if we don't have a server address.
+	if ( empty( $_SERVER['SERVER_ADDR'] ) ) {
+		return false;
+	}
+
+	// Set our transient key for use later.
+	$k  = 'hello_server_info_cache_' . sanitize_text_field( $_SERVER['SERVER_ADDR'] );
+
+	// If we don't want the cache'd version, delete the transient first.
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+		delete_transient( $k );
+	}
+
+	// It wasn't there, so regenerate the data and save the transient.
+	if ( false === $data = get_transient( $k )  ) {
```

Comment:
Just realized that the cache key is being changed based on `$_SERVER['SERVER_ADDR']`, but I'd still lean towards the `wp_cache_*()` series of functions over transients, to avoid any potential conflict if the cache ends up getting written to the database.